### DISCUSSION TITLE
Battery powered eufy camera.

### DISCUSSION THREAD

**Discussion by Disincentivato** [fuatakgun/eufy_security#1039] [Q&A]
![image](https://github.com/fuatakgun/eufy_security/assets/90531670/a649b98b-c255-4ed2-9fe0-80917b673a6d)
![image](https://github.com/fuatakgun/eufy_security/assets/90531670/99388e2d-0862-4cb2-8810-b7bf69717aa9)
Sometime during the stream the camera works sometime not. I have never viewed the last event image when the camera is not streaming. Are there solutions?
There are many problems with the view of battery powered cameras I see.
And as I wrote in another post, none of the camera have audio, is this normal?

**Comment by protomota** [comment]
I can't get any of my battery powered wireless Eufy Cam 3s to work either. If I trigger motion on the camera itself, it gets a live feed for a few seconds and starts the RTSP stream, but there doesn't seem to be a way to view the camera's RTSP stream without actually triggering motion on the camera itself, which is useless if you just want a live view of the camera at other times. I realize that these battery cameras should not have a constant RTSP stream, but I thought this plugin would allow for starting the stream with service: camera.turn_on, but I have not been able to get that to work. Is this maybe a bug with the Home Base 3? I have a Home Base 2 as well that I am going to set up to see if it works better until Home Base 3 functionality gets sorted out.

I am curious if you are also using a Home Base 3?

I also have some 2K indoor cameras which work fine, as I can see you also have other wired cameras that work as well.

**Comment by Disincentivato** [comment]
Hi, sorry for the late reply. I have home base 2... and yes, this integration is useless with battery powered camera.
